app/src/Containers/SearchContainer.tsx

```diff
@@ -294,7 +297,33 @@ const SearchContainer = () => {
     })
   }
 
-  const searchByArray = (
+  const exactSearchByArray = (
+    searchQueries: string[],
+    searchField: 'client' | 'description' | 'name' | 'role' | 'skills' | 'sector',
+  ): Projects => {
+    let results = [] as Projects
+
+    if (projects) {
+      results = projects.filter(
+        project => {
+          const anySearchQueriesMatching = searchQueries.some(searchQuery => {
+            if (typeof project[searchField] === "string") { // most fields are strings, but some are an array of strings (i.e. skills)
+              const isString = project[searchField] as string
```

Comment:
Just a small thing, but there's [a convention](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/) to often use `is` at the beginning of boolean variables, so it might be better to name `isString` something like `stringField` or `stringValue` instead

Reply:
Quite right, thanks Joe